pytorch_lightning/utilities/imports.py

```diff
@@ -25,18 +25,46 @@
 from pkg_resources import DistributionNotFound
 
 
+def _package_available(package_name: str) -> bool:
+    """Check if a package is available in your environment.
+
+    >>> _package_available('os')
+    True
+    >>> _package_available('bla')
+    False
+    """
+    try:
+        return find_spec(package_name) is not None
+    except AttributeError:
+        # Python 3.6
+        return False
+    except (ImportError, ModuleNotFoundError):
+        # Python 3.7+
+        return False
+
+
 def _module_available(module_path: str) -> bool:
-    """Check if a path is available in your environment.
+    """Check if a module path is available in your environment.
 
     >>> _module_available('os')
     True
```

Comment:
For checking package availability, one has to use `_package_available` above, right?
```suggestion
```

Reply:
It shall be optional